doc/spec/file-and-link-specifiers.md

```diff
@@ -0,0 +1,159 @@
+# `file:` and `link:` specifiers
+
+`specifier` refers to the value part of the `package.json`'s `dependencies`
+object.  This is a semver expression for registry dependencies and
+URLs and URL-like strings for other types.
+
+### Dependency Specifiers
+
+* A valid `link:` specifier is either an absolute path, (eg
+  `/path/to/thing`, `d:\path\to\thing`) or a relative path, eg
+  `../path/to/thing`, `path\to\subdir`).  `link:` specifiers do not
+  distinguish between forward and backslashes.  That is, we consider `link:a\b`
+  and `link:a/b` to be the same value.
+* Attempting to install a specifer that has a windows drive letter will
+  produce an error on non-Windows systems.
+
+* A valid `file:` specifier points is:
```

Comment:
Did you mean "A valid `file:` specifier also is" like in the next item? I am finding it difficult to understand this line.